pkg/test/integration/envtest.go

```diff
@@ -191,9 +191,9 @@ func (t *TestEnvironment) StartManager(ctx context.Context) error {
 func (t *TestEnvironment) WaitForWebhooks() {
 	port := env.WebhookInstallOptions.LocalServingPort
 
-	timeout := 1 * time.Second
+	timeout := 30 * time.Second
 	for {
-		time.Sleep(1 * time.Second)
+		time.Sleep(30 * time.Second)
```

Comment:
Let's keep 1s poll time with higher timeout to not spent additional time sleeping when port become open between iterations.

Reply:
+1, this wasn't up for a review yet, I was just testing the CI in the sync commit, haven't even reviewed after myself.